ddp-workspace/projects/ddp-sdk/src/lib/services/serviceAgents/submissionManager.service.spec.ts

```diff
@@ -90,11 +91,14 @@ describe('SubmissionManagerTest',() => {
     it('test isSubmissionInProgress$ observable', (done) => {
         // Fake the call to service agent, including the response delay
         const httpCallDelay = 500;
+
         serviceAgent.saveAnswerSubmission.and.callFake((): Observable<PatchAnswerResponse> => {
             return of({
                 answers: [],
                 blockVisibility: []
-            }).pipe(delay(httpCallDelay));
+            }).pipe(
+                delayWhen(() => timer(httpCallDelay))
```

Comment:
The only thing I changed in this file:
`delay(httpCallDelay)` => `delayWhen(() => timer(httpCallDelay)`.
Just a guess, because it is practically the same, 
but other tests in the file have `delayWhen` and don't fail.
[This particular test (line 145)](https://github.com/broadinstitute/ddp-angular/pull/1043/files#:~:text=expect(returnedInProgressValues%5B2%5D.delay).toBeGreaterThanOrEqual(httpCallDelay)%3B) fails from time to time.

Let's try. It it does not help, may be it makes sense to avoid/comment/remove this test/line as we are going to have a required tests check passed in every PR.